docs/contracts/using-for.rst

```diff
@@ -16,17 +16,18 @@ at contract level.
 
 The first part, ``A``, can be one of:
 
-- a list of file-level or library functions (``using {f, g, h, L.t} for uint;``) -
-  only those functions will be attached to the type.
-- the name of a library (``using L for uint;``) -
-  all functions (both public and internal ones) of the library are attached to the type
+- A list of file-level or library functions (``using {f, g, h, L.t} for uint;``) -
```

Comment:
```suggestion
- A list of file-level or library functions (e.g ``using {f, g, h, L.t} for uint;``) -
```
Using `uint` as `B` can be a bit confusing to the reader here, as we're not specifying explicitly that this is just an example. Same thing on line 22. Keep in mind that this is a nit.

Reply:
Good point. Changed.